tensorflow/core/kernels/mkl_tfconv_op.cc

```diff
@@ -29,8 +29,8 @@ limitations under the License.
 #include "tensorflow/core/util/tensor_format.h"
 
 #include "tensorflow/core/util/mkl_util.h"
-#include "third_party/mkl/include/mkl_dnn.h"
-#include "third_party/mkl/include/mkl_dnn_types.h"
+#include "mkl_dnn.h"
```

Comment:
previously all the headers were manually copied to third_paarty/mkl/include.
However, now we use "includes" field of the cc_library rule "intel_binary_blob", which adds the include folder for mkl to the gcc commands as a "-iquote" flag. Then all these libraries become available to include without the need for a relative path.